Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/AutocorrectionDisabledModifier.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AutocorrectionDisabledModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Sets whether to disable autocorrection for this view.
+///
+/// ```html
+/// <TextField modifiers={autocorrection_disabled(@native)}>Enter text</TextField>
+/// <TextField modifiers={autocorrection_disabled(@native, disable: false)}>Enter text</TextField>
+/// ```
+///
+/// ## Arguments
+/// * ``disable``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct AutocorrectionDisabledModifier: ViewModifier, Decodable {
+    /// A Boolean value that indicates whether autocorrection is disabled for this view. The default value is true.
+    #if swift(>=5.8)
+    @_documentation(visibility: public)
+    #endif
+    private var disable: Bool
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.disable = try container.decodeIfPresent(Bool.self, forKey: .disable) ?? true
```

Comment:
This can probably be simplified if the default is set in the Elixir schema:
```suggestion
        self.disable = try container.decode(Bool.self, forKey: .disable)
```